postgres/experiment.yaml

```
@@ -8,6 +8,9 @@ metadata:
     stormforge.io/objective: 'duration-vs-cost'
     stormforge.io/scenario: 'pgbench'
 spec:
+  optimization:
+  - name: experimentBudget
+    value: "400"
```

```suggestion
    value: "40"
```

Since this is an example and users probably don't expect this take days I would suggest a smaller budget so only a small number of trials are executed. We still have a recommended / optimial recommendation with 40 trials, right?